File: samples/JavaScript/Json Store/Session/Open Session.js
session.setAsync(false);


var s = "";
var app = darwino.jstore.createRemoteApplication("$darwino-jstore");

// There is a session that is available in the playground, and usable right away.
// This makes it easier for the playground samples
var user = session.getUser()
s += "Pre-created session, user="+user.getCn()+"\n";


// Outside of the playground, here is how a session using the currently authenticated user should be created:
var session2 = app.createSession(); session2.setAsync(false);
s += "Current user session, user="+session2.getUser().getCn()+"\n";

// An a session can also be created using a user/password but only when dealing with a different server
// so the basic authentication parameters are passed to the request
// In case of the same server, then the session authentication is used and the parameters are ignored
var session3 = app.createBasicSession("amass@triloggroup.com","password"); session3.setAsync(false);
s += "Al Mass session, user="+session3.getUser().getCn()+"\n";


darwino.Utils.setText("content","{0}",s);

File: samples/Java/Json Store/Session/Open Session.java
// Use whatever Application object you have here
// As the Playground is runninon top of IBM Domino, it creates a DarwinoDominoApplication
String s = "";
LocalJsonDBServer server = DarwinoDominoApplication.get().getLocalJsonDBServer();

// Create a new Session
User user = new UserImpl("james@007.com","James Bond",null,null);
Session session2 = server.createSession(user,null);
s += "Current user session, user="+session2.getUser().getCn()+"<br/>";

out.println(s);	
